middleware/index.ts

```diff
@@ -184,13 +200,29 @@ export default function applySfFxMiddleware(request: any, state: any, resultArgs
         throw new Error('Request Data not provided');
     }
 
+    // Logger is the last element in the array
+    const logger: Logger = resultArgs[resultArgs.length-1];
+    if (!logger) {
+        throw new Error('Logger not provided in resultArgs from node system function');
+    }
+
+    //use secret her in lieu of DEBUG runtime environment var until we haave deployment time support of config var
```

Comment:
her => here.
haave => have.

Reply:
good catch... errhh... blame on this macbook keyboard!